libs/scripts/delayed.js

```diff
@@ -69,8 +69,11 @@ const loadDelayed = ([
   loadScript,
   loadStyle,
   loadIms,
+  MILO_EVENTS,
 ], DELAY = 3000) => new Promise((resolve) => {
   setTimeout(() => {
+    const event = new Event(MILO_EVENTS.DELAYED);
+    document.dispatchEvent(event);
```

Comment:
As you don't re-use the event
```suggestion
    document.dispatchEvent(new Event(MILO_EVENTS.DELAYED));
```